### ISSUE TITLE
Mob heads lore

### ISSUE THREAD

**Issue by Sikatsu** [Rocologo/MobHunting#619]
Hmm, when they do [item] using a mobhunting head, they see a lot of random information. Not sure if that can be changed, so this is more like a question. Example:

![image](https://user-images.githubusercontent.com/5664135/77859662-0d34eb00-720b-11ea-9c8d-178bb4b2cd1b.png)

**Comment by Rocologo** [comment]
ProtocolLib Should Remove this?

**Comment by Sikatsu** [comment]
I do have protocollib installed. When I hover over the head in my inventory, only ''Mobhunting Reward'' is displayed, but when using [item] from the ShowItem plugin, the hidden stuff gets shown.

**Comment by Rocologo** [comment]
Hmm... its this a plugin or what do you do when you see this? I con only remove this if it is an action supported by ProtocolLib and this is hard to check

https://ci.dmulloy2.net/job/ProtocolLib/javadoc/overview-summary.html

**Comment by Rocologo** [comment]
This is the list I can choose from

https://ci.dmulloy2.net/job/ProtocolLib/javadoc/com/comphenix/protocol/PacketType.Play.Client.html

**Comment by Rocologo** [comment]
Or this one....

https://ci.dmulloy2.net/job/ProtocolLib/javadoc/com/comphenix/protocol/PacketType.Play.Server.html

**Comment by Sikatsu** [comment]
Yeah, its the plugin ShowItem, when you use [item] in the chat it will then display the hidden lines.
Not sure what else I can tell you, Im not a developer haha

**Comment by Rocologo** [comment]
I need to try the plugin on my test server to fully understand but I'm of to bed now.

**Comment by Sikatsu** [comment]
Understood

**Comment by Rocologo** [comment]
Its a paid plugin. I cant test this plugin.

**Comment by Rocologo** [comment]
You could try this in the ShowItem config.yml

  # Block items with certain strings in the lore from getting shown (case-insensitive)
  # r= prefix indicates that the string should be handled as a regex
  lore: []
  # - "name 1"
  # - "r=name \\d"

**Comment by Rocologo** [comment]
Or you could ask the guy to add support for mu plugin :-) It would probaly  be easy for him to remove the lines starting with "Hidden:"

**Comment by Sikatsu** [comment]
I will look into what I can do, or perhaps the developer of ShowItem. Will let you know, this is not too important anyway. Might be other issues on the github with a higher priority ^^

**Comment by Phoenix616** [comment]
The issue seems to be due to MobHunting not filtering out the hidden tags from the item component chat packet which ShowItem uses to display it. Filtering the PacketType.Play.Client.CHAT packet should be able to handle that.

**Comment by Momshroom** [comment]
I have protocollib and I see the “hidden” lines just hovering over many
types of mob heads in my inventory.

**Comment by Sikatsu** [comment]
Any update on this one?
